### PR TITLE
Release 2.7.3-beta1

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

Looking for the PR for the upcoming first beta of the v2.7.3 series? Well you've just found it, congratulations!

## Release checklist

- [x] ~~Check to see if there are any errors in Sentry that have only occurred since the last production release~~
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.7.3-beta1": [
+      "[New] Expand diffs to get more context about your changes - #11965",
+      "[New] Preliminary support for Windows ARM64 and Apple Silicon - #9034. Thanks @dennisameling!",
+      "[Added] Add a Hide Whitespace setting to both the Changes and History tabs - #6818. Thanks @say25!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Hide Whitespace in diffs on both Changes and History tabs - #6818. Thanks @say25!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+      "[Fixed] Delete branch on branches dropdown context menu incorrectly detected the branch as existing on the remote after being deleted remotely and fetch had occurred - #11923"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Show correct state of remote branch when deleting local branches - #11923"
```

**Review by billygriffin — DISMISSED** [review]
LET'S GOOOO

**Review by tidy-dev — APPROVED** [review]
:sparkle:

**Pull request merged**